lib/matplotlib/rcsetup.py

```diff
@@ -299,6 +301,12 @@ def validate_color(s):
 
     raise ValueError('%s does not look like a color arg%s' % (s, msg))
 
+def deprecate_axes_colorcycle(value):
+    warnings.warn("axes.color_cycle is deprecated.  Use axes.prop_cycle "
+                  "instead. Will be removed in 2.1.0")
+    return validate_colorlist(value)
```

Comment:
Ahh, okay, this just wraps around the validator... hmm, why do it this way then and not as a decorator for ``validate_colorlist``.

Also what do you think about setting ``axes.prop_cycle`` here?  That should make the logic easier elsewhere as we don't have to care about color_cycle elsewhere in the code...